### ISSUE TITLE
Username remains visible even after logout

### ISSUE THREAD

**Issue by madhurgupta10** [commons-app/apps-android-commons#3645]
**Summary:** 
Username (of the last logged in user) remains visible in the navigation drawer even after logout.

**Steps to reproduce:** 
Login > Logout > Open Navigation Drawer
 
**Commons app version:** 
betaDebug lastest master

**Comment by sivaraam** [comment]
Yeah. I observed this too.

**Comment by madhurgupta10** [comment]
@sivaraam I have noticed it for a while now, can someone confirm if it's intentional or a bug? My guess is that it is a bug.

**Comment by sivaraam** [comment]
> can someone confirm if it's intentional or a bug?

I would have no doubt about this being a bug. Showing the username when a user is logged out is just misleading and has no obvious benefit to me. So, I don't think this is intentional. Correct me if I'm wrong.

**Comment by madhurgupta10** [comment]
> > can someone confirm if it's intentional or a bug?
> 
> I would have no doubt about this being a bug. Showing the username when a user is logged out is just misleading and has no obvious benefit to me. So, I don't think this is intentional. Correct me if I'm wrong.

I agree, the reason why I suspect it because it has been there unnoticed for so long.

Anyways, I would be happy to fix it!

**Comment by sivaraam** [comment]
> Anyways, I would be happy to fix it!

Kindly do so! I'll assign this to you :)

**Comment by macgills** [comment]
this issue is irrelevant after #3649 is fixed, you shouldn't be able to see the navigation drawer after a logout.

**Comment by sivaraam** [comment]
> this issue is irrelevant after #3649 is fixed, you shouldn't be able to see the navigation drawer after a logout.

Well, we would see it if we 'Skip' the log-in. I was thinking that was the reason behind this issue? 🤔

![Screenshot_1586455460](https://user-images.githubusercontent.com/12448084/79228272-cdaf0580-7e7e-11ea-9b9c-76de917d68b4.png)

**Comment by macgills** [comment]
I skipped the login after checking out #3649 and couldn't see my name in the drawer

**Comment by madhurgupta10** [comment]
@macgills If you click on username area then can you launch achievements activity?

**Comment by macgills** [comment]
Yes as I said in my [comment](https://github.com/commons-app/apps-android-commons/pull/3648#issuecomment-613357495) on the PR where I asked you to open a separate issue

**Comment by madhurgupta10** [comment]
> Yes as I said in my [comment](https://github.com/commons-app/apps-android-commons/pull/3648#issuecomment-613357495) on the PR where I asked you to open a separate issue

@macgills I opened a new issue #3652 and it can be closed with #3648

**Comment by macgills** [comment]
Awesome
